gfx/video_driver.c

```diff
@@ -1099,6 +1107,16 @@ static bool video_driver_init_internal(bool *video_is_threaded)
    command_event(CMD_EVENT_OVERLAY_DEINIT, NULL);
    command_event(CMD_EVENT_OVERLAY_INIT, NULL);
 
+#ifdef HAVE_VIDEO_LAYOUT
+   video_layout_driver_init(video_driver_data, current_video->video_layout_render_interface(video_driver_data));
```

Comment:
why isn't this inside the if statement? we don't want to allocate memory we'll never use

Reply:
Reason is the interface is set at that point, if it were disabled on startup then enabled in menu, it'd crash. I'll see if there's a public interface to get video_driver_data and current_video

Reply:
Exposed the interface so now enable/disable connects directly to init/deinit